docs/README_cn.md

```diff
@@ -64,7 +64,7 @@ v0.11.0 版本已于 2021 年 1 月 31 日发布，可通过查阅 [更新日志
 | [Slowfast](/configs/recognition/slowfast/slowfast_r50_video_4x16x1_256e_kinetics400_rgb.py)|256p videos|Disk|8x8 | **[0.69](https://download.openmmlab.com/mmaction/benchmark/recognition/mmaction2/slowfast_256p_videos_disk_8x8.zip)** | x | x | [1.04](https://download.openmmlab.com/mmaction/benchmark/recognition/pyslowfast/pysf_slowfast_r50_4x16_video.log) |
 | [R(2+1)D](/configs/recognition/r2plus1d/r2plus1d_r34_video_8x8x1_180e_kinetics400_rgb.py)|256p videos |Disk| 8x8|**[0.45](https://download.openmmlab.com/mmaction/benchmark/recognition/mmaction2/r2plus1d_256p_videos_disk_8x8.zip)** | x | x | x |
 
-更多详情可见 [基准测试](docs/benchmark.md)
+更多详情可见 [基准测试](/docs/benchmark.md)
```

Comment:
The change possibly doesn't affect anything

Reply:
it matters, or you can try it : )

Reply:
oh the cn version